### PR TITLE
Add com.snowplowanalytics.snowplow.ecommerce/refund/jsonschema/1-0-0 …

### LINKED ISSUE #1259: Add com.snowplowanalytics.snowplow.ecommerce/refund/jsonschema/1-0-0
Add snowplow ecommerce refund schema.

### PR DESCRIPTION
**Pull request by igneel64**
Add ecommerce refund schema.

close #1259

### CODE REVIEW

File: schemas/com.snowplowanalytics.snowplow.ecommerce/refund/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            "minimum": 0.00,
+            "maximum": 9999999
+        },
+        "refund_reason": {
+            "type": [
+              "string",
+              "null"
+            ],
+            "description": "Reason for refunding the whole or part of the transaction.",
+            "maxLength": 256
```
Comment (emielver) [inline schemas/com.snowplowanalytics.snowplow.ecommerce/refund/jsonschema/1-0-0:36]: Should we allow the reason to be longer?

**Review by emielver — APPROVED** [review]
Just one question but otherwise looks good to me!

File: schemas/com.snowplowanalytics.snowplow.ecommerce/refund/jsonschema/1-0-0 [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+        "name": "refund",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "transaction_id": {
+            "description": "The ID of the transaction.",
```
Comment (matus-tomlein) [inline schemas/com.snowplowanalytics.snowplow.ecommerce/refund/jsonschema/1-0-0:13]: Nit: knowing nothing about ecommerce, I am just wondering if it's safe to assume that we also have the ID of the refunded transaction available or should we rather make it optional?

**Review by matus-tomlein — APPROVED** [review]
Agree with Emiel on making `refund_reason` max length longer, but otherwise LGTM!

**Comment by igneel64** [comment]
Cherry picked into R140 https://github.com/snowplow/iglu-central/pull/1256

**Pull request closed without merging**